logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -141,6 +165,57 @@ class ConversationDataSource(
     override suspend fun deleteMember(conversationID: QualifiedIDEntity, userID: QualifiedIDEntity): Either<CoreFailure, Unit> =
         wrapStorageRequest { conversationDAO.deleteMemberByQualifiedID(conversationID, userID) }
 
+    override suspend fun createGroupConversation(name: String, members: List<Member>, options: ConverationOptions): Either<CoreFailure, Conversation> = suspending {
+        wrapApiRequest {
+            conversationApi.createNewConversation(
+                CreateConversationRequest(
+                    if (options.protocol == ConverationOptions.Protocol.PROTEUS) members.map { idMapper.toApiModel(it.id) } else emptyList(),
+                    name,
+                    options.access.toList().map(conversationMapper::toApiModel),
+                    options.accessRole.toList().map(conversationMapper::toApiModel),
+                    userRepository.getSelfUser().firstOrNull()?.team?.let { ConvTeamInfo(false, it) },
+                    null,
+                    if (options.readReceiptsEnabled) 1 else 0,
```

Comment:
I think if we can replace this part with enum would be great! or at least have some doc comments to make sure we all know the meaning of these numbers.

Reply:
The backend decided to have an `Int` for this boolean property 🤷‍♂️ I added a comment for this property in the network module.

Reply:
But we can have an enum in our side that have the int values inside of it.

Reply:
Ok, done